x-pack/plugins/ml/public/services/ml_api_service/results.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// Service for obtaining data for the ML Results dashboards.
+
+import { pick } from 'lodash';
+import chrome from 'ui/chrome';
+
+import { http } from 'plugins/ml/services/http_service';
+
+const basePath = chrome.addBasePath('/api/ml');
+
+export const results = {
+  getAnomaliesTableData(obj) {
+    const data = pick(obj, [
```

Comment:
if you're passing through the whole object there's no need to use `pick`
The current uses of `pick` in `ml_api_service` are where the `obj` being passed in contains additional items which should not be part of the endpoint payload.